### ISSUE TITLE
Error using Trigger Tag "mailGraph.graph"

### ISSUE THREAD

**Issue by BernhardLinz** [moudsen/mailGraph#46]
I try to use the Tag `mailGraph.graph`, i got an error in the log:
```
2024-11-20 12:58:19 : !! (8) "Undefined index: graphid" at line #1394 of "/var/www/html/mailGraph/mailGraph.php"
2024-11-20 12:58:19 : !! (8) "Undefined index: name" at line #1395 of "/var/www/html/mailGraph/mailGraph.php"
2024-11-20 12:58:19 : # Adding FORCED graph #
2024-11-20 12:58:19 : !! (8) "Undefined index: graphtype" at line #1404 of "/var/www/html/mailGraph/mailGraph.php"
```
In the email i got the message: **Zabbix has received an incorrect request. Field "graphid" is not integer.**

I found the reason, there is an error in the `mailGraph.php` at Line 1373:
```
            $theGraph = $forceGraphInfo;
```
must be changed to
```
            $theGraph = $forceGraphInfo['result'][0];
```
then everything works fine.

Regards,
Bernhard

**Comment by moudsen** [comment]
I'm impressed and will take this enhancement forward in my next release (on short notice).
Appreciated!
Kind regards, Mark

**Comment by moudsen** [comment]
Fixed in v2.17
